charts/jenkins/VALUES_SUMMARY.md

```diff
@@ -81,16 +81,14 @@ The following tables list the configurable parameters of the Jenkins chart and t
 
 | Parameter                         | Description                          | Default                                   |
 | --------------------------------- | ------------------------------------ | ----------------------------------------- |
-| `controller.slaveListenerPort`        | Listening port for agents            | `50000`                                   |
-| `controller.slaveHostPort`            | Host port to listen for agents            | Not set                              |
-| `controller.slaveKubernetesNamespace` | Namespace in which the Kubernetes agents should be launched  | Not set           |
-| `controller.slaveDefaultsProviderTemplate` | The name of the pod template to use for providing default values | Not set  |
-| `controller.slaveJenkinsUrl`          | Overrides the Kubernetes Jenkins URL    | Not set                                |
-| `controller.slaveJenkinsTunnel`       | Overrides the Kubernetes Jenkins tunnel | Not set                                |
-| `controller.slaveConnectTimeout`      | The connection timeout in seconds for connections to Kubernetes API. Minimum value is 5. | 5 |
-| `controller.slaveReadTimeout`         | The read timeout in seconds for connections to Kubernetes API. Minimum value is 15. | 15 |
-| `controller.slaveListenerServiceType` | Defines how to expose the slaveListener service | `ClusterIP`                    |
-| `controller.slaveListenerLoadBalancerIP`  | Static IP for the slaveListener LoadBalancer | Not set                       |
+| `controller.agentListenerPort`        | Listening port for agents            | `50000`                                   |
+| `controller.agentHostPort`            | Host port to listen for agents            | Not set                              |
+| `controller.agentJenkinsUrl`          | Overrides the Kubernetes Jenkins URL    | Not set                                |
+| `controller.agentJenkinsTunnel`       | Overrides the Kubernetes Jenkins tunnel | Not set                                |
+| `controller.agentConnectTimeout`      | The connection timeout in seconds for connections to Kubernetes API. Minimum value is 5. | 5 |
+| `controller.agentReadTimeout`         | The read timeout in seconds for connections to Kubernetes API. Minimum value is 15. | 15 |
+| `controller.agentListenerServiceType` | Defines how to expose the agentListener service | `ClusterIP`                    |
```

Comment:
I'd vote to move all the values used to configure the kubernetes plugin under `agent` 
```yaml
agent:
  # URL for connecting to the Jenkins contoller
  jenkinsUrl:
  # connect to the specified host and port, instead of connecting directly to the Jenkins controller
  jenkinsTunnel:
  kubernetesConnectTimeout:
  kubernetesReadTimeout:
```

Reply:
That's a good idea. Let's do that in another PR. Do you want to do it?

Reply:
Yep. I'll open a PR.